org.fsnebula.Knossos.yml

```
@@ -0,0 +1,68 @@
+app-id: org.fsnebula.Knossos
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '5.15-21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+  - org.freedesktop.Sdk.Extension.rust-stable
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+command: knossos
+finish-args:
+  - --socket=x11
```

```suggestion
  - --socket=fallback-x11
```

since you claim wayland support